demos/OptimalPlanning.cpp

```diff
@@ -74,6 +75,7 @@ enum optimalPlanner
     PLANNER_FMTSTAR,
     PLANNER_BFMTSTAR,
     PLANNER_INF_RRTSTAR,
+    PLANNER_RRTSTARSMART,
```

Comment:
As described by the comment in the code, I think it is best to keep this list in alphabetical order.
If not, at least remove my comment.